.github/workflows/.test.yml

```
@@ -0,0 +1,561 @@
+# reusable workflow
+name: .test
+
+# TODO: hide reusable workflow from the UI. Tracked in https://github.com/community/community/discussions/12025
+
+on:
+  workflow_call:
+    inputs:
+      use_snapshotter:
+        required: true
+        type: boolean
+      graph_driver:
```

```suggestion
      storage_driver:
```

Since the flag is actually `--storage-driver`.

---

:facepalm: changed it!